packages/steiger/src/app.ts

```diff
@@ -69,3 +69,5 @@ export const linter = {
 export function defineConfig<Rules extends Array<Rule> = Array<Rule>>(config: Config<Rules>) {
   return config
 }
+
+export type Diagnostics = Awaited<ReturnType<typeof runRules>>
```

Comment:
suggestion: this type is essentially `Array<Diagnostic>`, where `Diagnostic` comes from `@steiger/types`. I'd suggest to refer to that type directly instead of getting the return type from `runRules`

Reply:
Ah, I didn't see that type – makes a lot of sense though. Thanks for the quick review!